server/utils.js

```diff
@@ -0,0 +1,21 @@
+// compares whether the req.params.id and authorized user Id match
```

Comment:
agian, this branch doesn't seem to have the auth updates that are on current master. please merge with master before committing